rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -183,6 +187,7 @@ public void save() throws IOException {
         feePerKbSupport.save();
         whitelistSupport.save();
         federationSupport.save();
+        lockingCapSupport.save(activations);
```

Comment:
as discussed, this `save` method shouldn't receive `activations` as a parameter, because the field would be part of locking cap support implementation.

See how this was implemented in the other supports, like in whitelist support impl for example:
https://github.com/rsksmart/rskj/blob/federation-support-refactor-integration/rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistSupportImpl.java#L29,
https://github.com/rsksmart/rskj/blob/federation-support-refactor-integration/rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistSupportImpl.java#L172

Reply:
Done!